bvm/ballerina-logging/src/main/java/org/ballerinalang/logging/util/LogConfigReader.java

```diff
@@ -37,14 +37,14 @@ public class LogConfigReader {
 
     public LogConfigReader() {
         BLogManager logManager = (BLogManager) LogManager.getLogManager();
-        InputStream configStream = this.getClass().getClassLoader().getResourceAsStream("logging.properties");
-        if (configStream != null) {
-            try {
+        try (InputStream configStream = this.getClass().getClassLoader().getResourceAsStream("logging.properties")) {
+            if (configStream != null) {
                 logManager.readConfiguration(configStream);
-            } catch (IOException e) {
+                configStream.close();
```

Comment:
```suggestion
```
We do not need to explicitly close. It will auto-close.

Reply:
Yes. Since it is a autoclosinble we do not need to close.